ignite/contrib/metrics/regression/canberra_metric.py

```diff
@@ -22,13 +25,21 @@ class CanberraMetric(_BaseRegression):
 
     """
 
+    def __init__(
+        self, output_transform: Callable = lambda x: x, device: Union[str, torch.device] = torch.device("cpu")
+    ):
+        self._sum_of_errors = None
+        super(CanberraMetric, self).__init__(output_transform, device)
+
+    @reinit__is_reduced
     def reset(self):
-        self._sum_of_errors = 0.0
+        self._sum_of_errors = 0
```

Comment:
Just remarked that this metric does not use torch tensor for accumulators. Can you update that too ? Take a look at Accuracy for an example

Reply:
@vfdev-5 done